webcompat/static/js/lib/wizard/steps/sub-category.js

```diff
@@ -36,7 +40,12 @@ const handleSubCategory = (event) => {
   notify.publish("showStep", { id: "confirmBrowser" });
 };
 
-radio.on("change", handleSubCategory);
+const initListeners = () => {
+  radio.on("change", handleSubCategory);
+  addKeyDownListeners(".step-subproblem");
```

Comment:
this is rather a nitpick, but perhaps you could pass `container` variable to addKeyDownListeners (since we already query dom for it at the top of each file), and then inside the function query for labels within that container. I'll leave this up to you though :)

Reply:
Ah! Yeah, that's a bit cleaner. Let me make the change. thxxx